webview/src/webview_common.cpp

```diff
@@ -104,7 +104,11 @@ static int Create(lua_State* L)
     info.m_Self = dmScript::Ref(L, LUA_REGISTRYINDEX);
     info.m_L = dmScript::GetMainThread(L);
 
-    int webview_id = Platform_Create(L, &info);
+    // added by dotGears/TrungB
```

Comment:
Please do not add these types of comments. You can use git blame to see who added/changed a line. And your contribution will be credited as well in the readme.

Reply:
oh ok, was my own company policy to quick find stuff.